migrations/20151202130049_user_reset_hash.js

```diff
@@ -0,0 +1,8 @@
+export async function up(knex, Promise) {
+  await knex.raw(`ALTER TABLE "users" ADD COLUMN reset_password_hash VARCHAR(40)`);
```

Comment:
if plan is to use strictly sha1 then `char(40)` should be used. otherwise, just use `text` without limitations (it doesn't cost you anything in postgres)